examples/funloc/funloc_params.yml

```diff
@@ -0,0 +1,138 @@
+# Todo:
+# - complete doc of params
+# - nest/categorize here
+tmin: -0.2
+tmax: 0.5
+t_adjust: -4e-3
+bmin: None
+bmax : 0.
+n_jobs: 1
+lp_cut: 40
+decim: 5
+proj_sfreq: 100
+n_jobs_mkl: 1
+n_jobs_fir: 1
+n_jobs_resample: 1
+filter_length: 'auto'
+drop_thresh: 0.
+epochs_type: 'fif'
+fwd_mindist: 0.005
+bem_type: '5120-5120-5120'
+auto_bad: None
+auto_bad_reject: None
+ecg_channel: None
+eog_channel: None
+plot_raw: False
+match_fun: None
+hp_cut: None
+cov_method: 'empirical'
+ssp_eog_reject: None
+ssp_ecg_reject: None
+baseline: (None, 0)
+reject_tmin: None
+reject_tmax: None
+lp_trans: 'auto'
+hp_trans: 'auto'
+movecomp: 'inter'
+sss_type: 'python'
+int_order: 8
+ext_order : 3
+tsss_dur: 60.
+st_correlation: 0.98
+trans_to: 'median'
+sss_origin: 'auto'
+fir_design: 'firwin'
+autoreject_thresholds: False
+autoreject_types: ('mag', 'grad', 'eeg')
```

Comment:
tuples in YAML source won't work, only lists. this will get imported as `"('mag', 'grad', 'eeg')"`